config/docker/kafka-env.properties

```diff
@@ -1,3 +1,2 @@
-#SPRING_KAFKA_CONSUMER_GROUP_ID=opfab
-SPRING_DESERIALIZER_KEY_DELEGATE_CLASS=org.apache.kafka.common.serialization.StringDeserializer
-SPRING_DESERIALIZER_VALUE_DELEGATE_CLASS=org.lfenergy.operatorfabric.cards.publication.kafka.consumer.CardCommandConsumerDeserializer
+#SPRING_KAFKA_CONSUMER_GROUP_ID=opfab-command
+SPRING_DESERIALIZER_VALUE_DELEGATE_CLASS=org.lfenergy.operatorfabric.cards.publication.kafka.consumer.KafkaAvroWithoutRegistryDeserializer
```

Comment:
Zou je de SPRING_DESERIALIZER_VALUE_DELEGATE_CLASS hier niet uit-commenten (is dat Nederlands?)
Anders overruled deze altijd de deserializer uit de .yml file.